packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/simulationservice/SimulationRequestController.java

```diff
@@ -201,6 +201,12 @@ public ResponseEntity<JobResponse> makeEnsembleCalibrateCiemssJob(
 				.getBody());
 	}
 
+	@GetMapping("ciemss/cancel/{id}")
+	@Secured(Roles.USER)
+	public ResponseEntity<JobResponse> cancelCiemssJob(@PathVariable("id") final UUID id) {
+		return ResponseEntity.ok(simulationCiemssServiceProxy.cancelJob(id).getBody());
```

Comment:
Is it worth it to try to catch a `FeignClientException` thrown by the proxy here? I can foresee some strange situation where the front end tries to cancel a job that is already done or has already been cancelled and I imagine the ciemss service would throw an exception in this case

Reply:
Thank you for the tip, will do!